addons/core/functions/fnc_getLrHalfDuplexOverride.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getLrHalfDuplexOverride
+
+    Author(s):
+        MorpheusXAUT
+
+    Description:
+        Returns the passed LR radio's capabilities to use full-duplex mode although global half-duplex has been enabled.
+
+    Parameters:
+        0: ARRAY - Radio
+            0: OBJECT - Radio object
+            1: STRING - Radio ID
+
+    Returns:
+        BOOLEAN - Half-duplex mode should be skipped for this SW radio
+
+    Example:
+        (call TFAR_fnc_activeLrRadio) call TFAR_fnc_getLrHalfDuplexOverride;
+*/
+
```

Comment:
Yeah. but the problem is if you pass an array to params and params only accepts type ARRAY.
It will check if the first entry in that Array is the datatype it wants.
So instead of 
`(call TFAR_fnc_activeLrRadio) call TFAR_fnc_getLrHalfDuplexOverride;`
You would need to
`[(call TFAR_fnc_activeLrRadio)] call TFAR_fnc_getLrHalfDuplexOverride;` Which is ugly.

If you have more parameters it's fine because you need to
`[(call TFAR_fnc_activeLrRadio), moreParameters] call TFAR_fnc_getLrHalfDuplexOverride;`
anyway.
So you essentially can't use params for functions that get the LR Radio directly.